libs/beacon/src/beacon_setup_service.cpp

```diff
@@ -329,6 +328,12 @@ uint64_t BeaconSetupService::PreDKGThreshold()
   return ret;
 }
 
+uint32_t BeaconSetupService::QualSize()
+{
+  // Set to 2/3n for now
```

Comment:
Won't this fall over if the threshold is higher than this?

Reply:
Yes, this is true. There should be a check that the QualSize is larger than threshold. Unfortunately this choice of qual size was a bit arbitrary so we don't have a nice analytical expression for it in terms of the threshold.